runner/fixture.js

```diff
@@ -20,6 +20,7 @@ import { window, document } from "../src/globals";
 
 		var fixture = document.getElementById( "qunit-fixture" );
 		if ( fixture ) {
+			fixture.setAttribute( "role", "region" );
```

Comment:
What is this one for? I would expect the fixture to be treated as a hidden thing and not a jump target. At any given moment for a developer consuming the test report or watching the progress, this wouldn't be meaningful or change too rapidly, I think?

I'm open to adding it, but want to understand why so that the intention can be preserved and not accidentally removed in the future.

I do worry that it might also be a compat concern, I'll have to do some digging to confirm that. The fixture should be as plain as possible so that test code has full control over what it behaves like. I can see how a frontend library would behave differently if placed inside a certain role area, or that e.g. a frontend library's test suite asserts one of its creations in the fixture to have an ancestor of role=…something which could now false-positive on the fixture itself.

Reply:
That's a really good point of incompatibility; I could see this holding off until 3.0, if it is in fact something we need.

This was due to #3 on the checklist: "All page content must be contained by landmarks".
I tagged the "main" qunit div as role=main, but the fixture is a sibling of that, not a child, so it needed its own role as well.

I dabbled with the idea of leaving some of this to the test writer - they maintain their HTML runners, and could just as well provide the right roles. The fix here could arguably just a doc update at best?

Reply:
I would not consider the fixture as part of the presented "page content".

Reply:
Reverted!